### PR TITLE
s/master/main/ and s/master/v1/ where applicable

### PR DESCRIPTION
**Pull request by krancour**
This PR straightens out links that break as a result of v1 moving from the master branch to the v1 branch.

### CODE REVIEW

File: docs/content/topics/developers.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ $ git clone https://github.com/brigadecore/brigade $GOPATH/src/github.com/brigad
 $ cd $GOPATH/src/github.com/brigadecore/brigade
 ```
 
-**Note**: this leaves you at the tip of **master** in the repository where active development
+**Note**: this leaves you at the tip of **main** in the repository where active development
```
Comment (vdice) [inline docs/content/topics/developers.md:63]: Do we want to get even more explicit here and mention `where active development on Brigade v2.x is happening`?  And/or offer the reminder that the tip of v1 development (even though about to be in maintenance mode) can be checked out via the `v1` branch.

File: docs/readme.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Brigade Docs
 
-* Browse the [docs on GitHub](https://github.com/brigadecore/brigade/tree/master/docs/content)
+* Browse the [docs on GitHub](https://github.com/brigadecore/brigade/tree/v1/docs/content)
 * Browse the [docs website at docs.brigade.sh](https://docs.brigade.sh)
```
Comment (vdice) [inline docs/readme.md:4]: This URL should be updated to `https://v1--brigade-docs.netlify.app/`, right?

File: docs/content/topics/developers.md (hunk 1 above)
Comment (krancour) [reply docs/content/topics/developers.md:63]: I started out on that path, initially, and then second guessed it. My worry was that I'd rather not promote unsolicited investment in v1. Any thoughts?

Comment (vdice) [reply docs/content/topics/developers.md:63]: sgtm 👍

File: docs/readme.md (hunk 2 above)
Comment (krancour) [reply docs/readme.md:4]: Good catch.

Comment (krancour) [reply docs/readme.md]: Fixed. And fyi, longer term, I will also work out a nicer URL than `https://v1--brigade-docs.netlify.app/`.

**Review by vdice — APPROVED** [review]

**Pull request merged**